debian/vzlogger.postinst

```diff
@@ -2,8 +2,14 @@
 
 set -e
 
+ADDUSER_VER=$(dpkg-query --showformat='${Version}' --show adduser)
+if dpkg --compare-versions "$ADDUSER_VER" lt 3.131; then
+    FORCE=" --force-badname "
```

Comment:
it would be cleaner to provide a default value, $FORCE might exist in the environment already?

Reply:
also, the space as part of the variable is pointless.